src/comp/comp.py

```diff
@@ -24,19 +24,19 @@ def __repr__(self):
 # Write a list comprehension that creates a list of names of everyone
 # whose name starts with 'D':
 print("Starts with D:")
-a = [i for i in humans if i.name[0] == 'D']
+a = [i.name for i in humans if i.name[0] == 'D']
 print(a)
 
 # Write a list comprehension that creates a list of names of everyone
 # whose name ends in "e".
 print("Ends with e:")
-b = [i for i in humans if i.name[-1] == 'e']
+b = [i.name for i in humans if i.name[-1] == 'e']
 print(b)
 
 # Write a list comprehension that creates a list of names of everyone
 # whose name starts with any letter between 'C' and 'G' inclusive.
 print("Starts between C and G, inclusive:")
-c = [char for char in humans if ord(char.name[0]) in range(ord('C'), ord('G') + 1)]
+c = [char.name for char in humans if ord(char.name[0]) in range(ord('C'), ord('G') + 1)]
```

Comment:
Awesome job dude!
https://www.programiz.com/python-programming/methods/built-in/ord